webpack.config.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+
+module.exports = {
+    mode: 'development',
+    entry: './src/appEntry.js',
+    output: {
+        path: path.resolve(__dirname),
+        filename: './App.js'
+    },
+    module: {
+        rules: [
+          {
+            test: /\.(scss)$/,
+            use: [
+              {
+                // Adds CSS to the DOM by injecting a `<style>` tag
+                loader: 'style-loader'
+              },
+              {
+                // Interprets `@import` and `url()` like `import/require()` and will resolve them
+                loader: 'css-loader'
+              },
+              {
+                // Loader for webpack to process CSS with PostCSS
+                loader: 'postcss-loader',
+                options: {
+                  plugins: function () {
+                    return [
+                      require('autoprefixer')
+                    ];
+                  }
+                }
+              },
+              {
+                // Loads a SASS/SCSS file and compiles it to CSS
+                loader: 'sass-loader'
+              }
+            ]
+          }
+        ]
+      }
+}
```

Comment:
Keep an eye out for missing newlines at EOF. It is good practice to have them, and GitHub will helpfully warn you with this icon.